graylog2-server/src/main/java/org/graylog2/plugin/configuration/ConfigurationRequest.java

```diff
@@ -113,6 +115,12 @@ public void check(Configuration configuration) throws ConfigurationException {
                         }
                         break;
                     case TextField.FIELD_TYPE:
+                        break;
```

Comment:
This breaks the `TextField.FIELD_TYPE`. Before this PR, the `TextField` and `DropdownField` used the same implementation. (by relying on the fallthrough property of the switch case) This needs to stay the way it was before.